app/forms/income/NonTurnoverIncomeAmountFormProvider.scala

```diff
@@ -14,19 +14,23 @@
  * limitations under the License.
  */
 
-package forms
+package forms.income
 
 import forms.mappings.Mappings
-import javax.inject.Inject
 import play.api.data.Form
 
+import javax.inject.Inject
+
 class NonTurnoverIncomeAmountFormProvider @Inject() extends Mappings {
 
-  def apply(): Form[BigDecimal] =
+  def apply(isAgentString: String, tradeName: String): Form[BigDecimal] =
     Form(
       "value" -> bigDecimal(
-        "nonTurnoverIncomeAmount.error.required",
-        "nonTurnoverIncomeAmount.error.nonNumeric")
-          .verifying(inBigDecimalRange(0, 100000000000.00, "nonTurnoverIncomeAmount.error.outOfRange")) //TODO amount verification inline with ticket 5553
+        s"nonTurnoverIncomeAmount.error.required.$isAgentString",
+        s"nonTurnoverIncomeAmount.error.nonNumeric.$isAgentString",
+        Seq(tradeName))
```

Comment:
Where is the tradeName going to be used? When we provide it as args here, is it not usually used for an input value in the messages file via `..{..}..`?

Reply:
This will be removed now